internal/web_server/web/handlers/dashboard_pages_handlers.go

```diff
@@ -165,31 +166,48 @@ func (dph *DashboardPagesHandler) accessSettings(c echo.Context) error {
 func (dph *DashboardPagesHandler) userDetails(c echo.Context) error {
 	uuid_id, err := uuid.Parse(c.QueryParam("user-id"))
 	if err != nil {
-		return echo.NewHTTPError(http.StatusBadRequest, "Invalid or missing user id")
+		return echo.NewHTTPError(http.StatusBadRequest, "Ugyldig eller manglende user-id")
```

Comment:
Burde det kanskje stå bruker ID i stedet for for user-id, som er engelsk? Fordi hvis vi skal vise det til brukeren sier ikke user-id noe til de.

Reply:
uenig, query parameter er `user-id`